py/packages/corpora_cli/commands/corpus.py

```diff
@@ -113,12 +113,18 @@ def sync(ctx: typer.Context):
         ).getvalue()
         c.console.print(f"Tarball created: {len(tarball)} bytes")
 
+        # TODO: there is a bug in ninja here, I think
```

Comment:
TODO: I think this might be a legit bug in ninja + openapi generation that we should report .. see the serverside for details.